releng/raw-binaries/Dockerfile

```diff
@@ -1,10 +1,54 @@
 ARG GO_VERSION
 FROM golang:${GO_VERSION}
 
+# Install system dependencies + cross build system dependencies
 RUN apt-get update && apt-get install -y --no-install-recommends \
-  jq \
+    asciidoc \
```

Comment:
Curious to know what `asciidoc` is being used for

Reply:
asciidoc is used for compiling `man` pages